wordle.py

```diff
@@ -74,11 +107,46 @@ def play(word):
 
         if guess.lower() == word:
             print('WOW YOU WON')
-            return
-    print("SORRY YOU LOST. The WORD was", word)
+            return 1
+    print("SORRY YOU LOST.")
 
 
 if __name__ == '__main__':
-    random_word = get_random_word("words.txt")
-    print(random_word)
-    play(random_word)
+    print("Starting...")
+    CHOSEN_WORD = get_random_word("words.txt")
```

Comment:
decide at this stage whether we have enough information for hints using a lightweight dictionary.

Reply:
We have updated the word list to include only those words for which we have hints.